src/commands/build.ts

```diff
@@ -110,6 +110,11 @@ export class BuildCommand implements Command {
       type: String,
       description: 'Updates the <base> tag if found in the entrypoint document.'
     },
+    {
+      name: 'npm',
+      type: Boolean,
+      description: 'Look for dependencies in "node_modules/".'
```

Comment:
polyserve is documented like this: Sets npm mode: component directory is "node_modules" and the package name is read from package.json

Since this also effects package name in `polymer serve`, I'd go with this description.

Reply:
Ok, does anything need to read the package name in build?